UI/Web/src/app/series-detail/series-detail.component.ts

```diff
@@ -244,30 +245,47 @@ export class SeriesDetailComponent implements OnInit, OnDestroy {
     });
   }
 
-	onScroll(): void {
-		const tabs = document.querySelector('.nav-tabs') as HTMLElement | null;
-		const main = document.querySelector('.main-container') as HTMLElement | null;
-		const content = document.querySelector('.tab-content') as HTMLElement | null;
-		let mainOffset = main!.offsetTop;
-		let tabOffset = tabs!.offsetTop;
-		let contentOffset = content!.offsetTop;
-		let mainScrollPos = main!.scrollTop;
-		
-		if (!document.querySelector('.nav-tabs.fixed') && (tabOffset - mainOffset) <= mainScrollPos) {
-			tabs!.classList.add("fixed");
-			tabs!.style.top = mainOffset+'px';
-		} else if (document.querySelector('.nav-tabs.fixed') && mainScrollPos <= (contentOffset - mainOffset)) {
-			tabs!.classList.remove("fixed");
-		}
-	}
-
-
-
-	get ScrollingBlockHeight() {
-		if (this.scrollingBlock === undefined) return 'calc(var(--vh)*100)';
-		const mainOffset = this.scrollingBlock.nativeElement.offsetTop;
-		return 'calc(var(--vh)*100 - ' + mainOffset + 'px)';
-	}
+  onScroll(): void {
+    const main = document.querySelector('.main-container') as HTMLElement;
+    if (main === null) return;
+
+    const info = document.querySelector('.info-container') as HTMLElement;
+    const navbar = document.querySelector('.navbar') as HTMLElement;
+    const mainScrollPos = main.scrollTop;
+    const infoHeight = info.offsetHeight + 16; //16px for margin
+    const companionHeight = this.companionBar!.nativeElement.offsetHeight;
+    const navbarHeight = navbar.offsetHeight;
+    const tabs = document.querySelector('.nav-tabs') as HTMLElement;
+
+    if (!document.querySelector('.nav-tabs.fixed') && (infoHeight) <= mainScrollPos) {
+      const mainOffset = main.offsetTop; //For mobile tab placement calc
+      const totalHeight = companionHeight + navbarHeight + 20; //For desktop tab placement calc
+      const vw = Math.max(document.documentElement.clientWidth || 0, window.innerWidth || 0);
+      const tabsWidth = tabs.clientWidth;
+      tabs.classList.add("fixed");
+      tabs.style.width = tabsWidth+'px'; //Stretch tab width across section
+      if (vw < 780) {
+        //Use different offset for mobile
+        tabs.style.top = mainOffset+'px';
+      } else {
+        tabs.style.top = totalHeight+'px';
+      }
+    } else if (document.querySelector('.nav-tabs.fixed') && mainScrollPos <= (infoHeight)) {
+      tabs.classList.remove("fixed");
+      tabs.style.width = 100+'%'; //set tab width back
+    }
+  }
+
+  get ScrollingBlockHeight() {
+    if (this.scrollingBlock === undefined) return 'calc(var(--vh)*100)';
+    const navbar = document.querySelector('.navbar') as HTMLElement;
+    if (navbar === null) return 'calc(var(--vh)*100)';
+    
+    const companionHeight = this.companionBar!.nativeElement.offsetHeight;
+    const navbarHeight = navbar.offsetHeight;
+    const totalHeight = companionHeight + navbarHeight + 21; //21px to account for padding
+		return 'calc(var(--vh)*100 - '+totalHeight+'px)';
```

Comment:
Fix this indentation

Reply:
Resolved with https://github.com/Kareadita/Kavita/pull/1322/commits/fe89668af0f9ab7ccbd114412a907fbfd08612b7